Cargo.toml

```diff
@@ -13,12 +13,34 @@ repository = "https://github.com/shssoichiro/ffmpeg-the-third"
 keywords = ["ffmpeg", "multimedia", "video", "audio"]
 categories = ["multimedia"]
 
-# Exclude CI scripts etc. from package
-exclude = [".github"]
+include = [
+    "LICENSE",
+    "Cargo.lock.MSRV",
```

Comment:
It probably doesn't make sense to include the `Cargo.lock.MSRV`, since the `Cargo.lock` doesn't get included either, and it's not needed for building this as a library from crates.io because upstream Cargo.lock files are ignored when building as a library dep.